src/main/java/core/basesyntax/service/impl/ReportMakerServiceImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportMakerService;
+
+public class ReportMakerServiceImpl implements ReportMakerService {
+    private static final StringBuilder REPORT = new StringBuilder("fruit,quantity\n");
```

Comment:
does it make sense to make StringBuilder a constant? if you try to generate the report twice, will it be correct?